Kickstarter-iOS/TestHelpers/MockAppEnvironment.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
```

Comment:
Previously this was part of the test file that I touched, so I've extracted this since I was doing the same work for `MockFeedbackGenerator` (keeps the test file small and readable)